### PR TITLE
7383_Tyan_6_3

### PR DESCRIPTION
**Pull request by Katty57**

### CODE REVIEW

**Comment by kulikovia** [comment]
Поясните, пожалуйста, выбор последовательности слоев модели?

**Comment by Katty57** [comment]
Был выбран данный порядок слоев по примеру построения сверточных нейронных сетей https://keras.io/examples/mnist_cnn/ , предложенный в задании практической работы. Данный порядок слоев показал самый лучший результат.

**Comment by Katty57** [comment]
> Поясните, пожалуйста, выбор последовательности слоев модели?
Был выбран данный порядок слоев по примеру построения сверточных нейронных сетей https://keras.io/examples/mnist_cnn/ , предложенный в задании практической работы. Данный порядок слоев показал самый лучший результат.

**Comment by kulikovia** [comment]
Код модели полностью идентичен приведенному в примере. Поэтому можете пояснить, какие параметры модели Вы бы попробовали модифицировать для улучшения результатов (если улучшение возможно)?

**Comment by Katty57** [comment]
> Код модели полностью идентичен приведенному в примере. Поэтому можете пояснить, какие параметры модели Вы бы попробовали модифицировать для улучшения результатов (если улучшение возможно)?

Был добавлен слой Dropout(0.25), который "борется" с переобучением. Также был добавлен слой Dense(128, actiovation='relu'). Были построены графики точности и ошибки.
image
image
image
image

Графики для модели, представленной в коде:
image
image

**Comment by kulikovia** [comment]
Спасибо. 2,5 балла

**Comment by Katty57** [comment]
> Спасибо. 2,5 балла

Спасибо! Извиняюсь, что беспокою, но у меня в ведомости на сайте не выставлены баллы за 5 практическую работу.
https://github.com/Ksenox/INS_2020/pull/111

С уважением, Екатерина Тян

**Comment by kulikovia** [comment]
Извините, работу нашел, она принята, оценку в ведомость добавил.
